app/etl/item/content/parsers/no_content.rb

```diff
@@ -10,14 +10,23 @@ def schemas
       external_content
       generic
       homepage
+      organisations_homepage
       person
       placeholder_corporate_information_page
+      placeholder_ministerial_role
+      placeholder_organisation
+      placeholder_policy_area
+      placeholder_topical_event
+      placeholer_working_group
+      placeholder_world_location
```

Comment:
This is probably a bigger question, but these look to me like document types, rather than the names of schemas (e.g. I can't see a `placeholder_world_location` schema, I imagine it just uses the `placeholder` schema)?